test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1294,14 +1294,16 @@ private static synchronized void showUI() {
 
     /**
      * Forcibly pass the test.
-     * <p>The sample usage:
-     * <pre><code>
-     *      PrinterJob pj = PrinterJob.getPrinterJob();
-     *      if (pj == null || pj.getPrintService() == null) {
-     *          System.out.println(""Printer not configured or available.");
-     *          PassFailJFrame.forcePass();
-     *      }
-     * </code></pre>
+     * <p>
+     * Use this method in semi-automatic tests when
+     * the test determines that all the conditions for passing the test are met.
+     * <p>
+     * <strong>Do not use</strong> this method in cases where a resource is unavailable or a
+     * feature isn't supported, throw {@code jtreg.SkippedException} instead.
+     *
+     * <p>A sample usage can be found in
+     * <a href="https://github.com/openjdk/jdk/blob/7283c8b/test/jdk/java/awt/FileDialog/SaveFileNameOverrideTest.java#L84">{@code
+     * SaveFileNameOverrideTest.java}</a>
```

Comment:
Javadoc update LGTM. Maybe the doc for forceFail() can be updated similarly?
@aivanov-jdk What do you suggest?

It can be done as a part of this PR change or a separate PR. Since it is a similar doc change, I think it can be done as part of this PR by the changing the JBS title appropriately  - _Update javadocs for PassFailJFrame's forcePass() & forceFail() methods_

```
    /**
     *  Forcibly fail the test.
     */
    public static void forceFail() {
        forceFail("forceFail called");
    }
```

Reply:
Absolutely!

I don't want to include any changes for `forcePass` here because they're unrelated to the problem we're addressing.

[The description](https://bugs.openjdk.org/browse/JDK-8355515#description) of JDK-8355515 explicitly mentions the intent:

> The description of `forceFail()` should be expanded, too.

Reply:
@honkar-jdk JBS title could be updated… I'm still against doing so.

Currently, `forcePass` provides an example that shouldn't be followed. It's a problem.

Expanding the docs for `forcePass` and `forceFail` would be a good addition, and I've submitted [JDK-8355515](https://bugs.openjdk.org/browse/JDK-8355515) to track this work.

As I mentioned in the description of JDK-8355515, adding a section in the main javadoc of `PassFailJFrame` would be even better. Although a new section doesn't eliminate expanding documentation for the methods themselves.

Reply:
Sounds good to me if forceFail() docs is going to be handled as separate issue.